packages/@react-spectrum/actionbar/src/actionbar.css

```diff
@@ -0,0 +1,39 @@
+.spectrum-ActionBar {
+  --spectrum-padding: var(--spectrum-global-dimension-size-100);
+  --column-gap: var(--spectrum-actionbuttongroup-text-button-gap-x);
+  --row-gap: var(--spectrum-actionbuttongroup-text-button-gap-y);
+
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  isolation: isolate;
+  justify-content: space-between;
+  padding: var(--spectrum-padding);
+
+  &:focus {
+    outline: none;
+  }
+
+  &.spectrum-ActionBar--primary {
+    background-color: var(--spectrum-actionbar-primary-background-color);
+  }
+
+  &.spectrum-ActionBar--secondary {
+    background-color: var(--spectrum-actionbar-secondary-background-color);
+  }
+
+  &.spectrum-ActionBar--warning {
+    background-color: var(--spectrum-actionbar-warning-background-color);
+  }
+
+  .spectrum-ActionBar__leading {
```

Comment:
`react-spectrum-ActionBar--leading` is the class targeting the left side of actionbar. This class goes with the div containing the close button and the text. Since it looked like you peepz use BEM and that div represents an element in the component, I just went with the double underscore notation. Maybe, since you know the context for the class, you might have a better name?

Reply:
ah, then it's just `react-spectrum-ActionBar-leading`
we almost use BEM